src/trace/step-function-service.ts

```diff
@@ -137,7 +137,11 @@ export class StepFunctionContextService {
       PARENT_ID,
     );
     const sampleMode = SampleMode.AUTO_KEEP;
-    const _DatadogSpanContext = require("dd-trace/packages/dd-trace/src/opentracing/span_context");
+    function getDatadogSpanContext() {
```

Comment:
Sorry, this might not cut it, we want to have a try-catch just like the wrapper method which does the same thing, I can push the change tomorrow morning if you want

Reply:
Sure. Thanks! I guess you are worried about the customers may miss the tracer. I'm not familiar with the idea of missing tracer. Shouldn't tracer alway be included in this layer?